.yamllint

```diff
@@ -11,3 +11,4 @@ rules:
     indent-sequences: false
   line-length:
     max: 120
+  trailing-spaces: false
```

Comment:
Instead of adding a rule that ignores trailing spaces across the board, I think it's preferable to just add the aws_instance_types.yml to the ignore list above, since it's autogenerated.

Reply:
done